browserify/browserify.d.ts

```diff
@@ -1,41 +1,66 @@
-// Type definitions for Browserify
-// Project: http://browserify.org/
-// Definitions by: Andrew Gaspar <https://github.com/AndrewGaspar/>
```

Comment:
please keep original author names. like this https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/jquery/jquery.d.ts#L3